phoenix-core/src/it/java/org/apache/phoenix/end2end/IndexExtendedIT.java

```diff
@@ -136,6 +136,9 @@ public void testMutableIndexWithUpdates() throws Exception {
         if (!mutable) {
             return;
         }
+        if (!localIndex && useSnapshot) {
```

Comment:
Is this an unrelated change?

Reply:
No, it is not related and I will remove it